<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mafOperationWidget</class>
 <widget class="QWidget" name="mafOperationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>319</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>231</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="boxOperation">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Operation Name</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>29</y>
         <width>201</width>
         <height>221</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutBox">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>199</width>
             <height>219</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_3">
            <property name="geometry">
             <rect>
              <x>-1</x>
              <y>-1</y>
              <width>201</width>
              <height>221</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayoutOperation"/>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonOkCancel">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>mafOperationWidget</receiver>
   <slot>execute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>mafOperationWidget</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>mafOperationWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>mafOperationWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>execute()</slot>
  <slot>cancel()</slot>
 </slots>
</ui>
